src/core/constants.js

```diff
@@ -97,3 +101,28 @@ export const acceptedInstallTypes = [
   UNINSTALL_COMPLETE,
   INSTALL_ERROR,
 ];
+
+// Tracking categories.
+export const INSTALL_CATEGORY = 'AMO Addon / Theme Installs';
+export const UNINSTALL_CATEGORY = 'AMO Addon / Theme Uninstalls';
+export const CLICK_CATEGORY = 'AMO Addon / Theme Clicks';
+
+// Error used to know that the setEnable method on addon is
+// not available.
+export const SET_ENABLE_NOT_AVAILABLE = 'SET_ENABLE_NOT_AVAILABLE';
+
+// Add-on statuses for mozAddonManager events.
+export const globalEventStatusMap = {
```

Comment:
It would seem that way. Just looked up https://github.com/mozilla/addons-frontend/pull/543 which appears to be where they came from.

Not sure if @muffinresearch had a reason in mind to lowercase them but I didn't comment on it. It looks like the strings are uppercase but the ones that are arrays or objects are camelCase.